### PR TITLE
Research: inverse design of plasmonics 

### PR DESCRIPTION
**Pull request by clpetix**

### CODE REVIEW

**Comment by clpetix** [comment]
@mayukh33 @pgkovakas @mzf0069 can you all give this a read/review?

**Comment by mayukh33** [comment]
Looks good to me!

File: _data/research.yaml [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
-  subtitle:
+- title:  Multi-scale inverse design of nanocrystal superlattices 
+  subtitle: 
   group: featured
-  image: images/software.png
-  link: 
-  description: 
+  image: images/inverse_design_figure.png
+  link: research/multiscale-inverse-design
+  description: This research engineers a novel computational approach for creating nanocrystal superlattices with controllable properties.
```
Comment (clpetix) [inline _data/research.yaml:15]: Remove period to be consistent with Mayukh

**Review by mayukh33 — APPROVED** [review]

**Review by mzf0069 — APPROVED** [review]

**Review by pgkovakas — APPROVED** [review]

**Review by mphoward — APPROVED** [review]
Thank you Levi! I reorganized this into the format I created on Mayukh's PR, and I made some edits to the text. Nice work!

**Pull request merged**